### DISCUSSION TITLE
CachyOS

### DISCUSSION THREAD

**Discussion by Medakekw** [prasanthrangan/hyprdots#1762] [General]
Did anyone try to install this on cachyos?
I mean installing cachyos minimal without a de and then running the install script like you can on endeavour os.

**Comment by kRHYME7** [comment]
@Medakekw  As long as it uses pacman, bigger chance it works

**Reply by Medakekw** [reply]
I tried doing a minimal install of cachyos(no desktop enviroment, just necessary stuff) and I got A LOT of conflicting packages which usually isnt a problem as I can open another session, manually resolve them and continue with the install but there were quite a lot... Would take me hours to fix them all

**Reply by kRHYME7** [reply]
just use ` ./install.sh -r ` to install the configs.
